src/main/java/com/answerdigital/answerking/exception/util/RestResponseEntityExceptionHandler.java

```diff
@@ -25,11 +25,12 @@
 @RestControllerAdvice
 public class RestResponseEntityExceptionHandler {
 
+    @SneakyThrows
```

Comment:
I don't think SneakyThrows is necessary here, or anywhere else it is used within this class!